### ISSUE TITLE
Recently getting tcp error

### ISSUE THREAD

**Issue by StevenHickson** [PromyLOPh/pianobar#515]
Recently I've been getting errors like this:  
TCP connection to audio-dc6-t1-1-v4v6.pandora.com:80 failed: Connection timed out  
/!\ Unable to open audio file (Input/output error)
Then it will hang until I get a new playlist. My internet is fine and I've tested it with other things so I'm not sure how/why this is happening.

**Comment by PromyLOPh** [comment]
Do you have IPv6 connectivity? #501 looks similar.

**Comment by StevenHickson** [comment]
So I looked at that earlier but I have no other problems with my connection at all. I did extensive testing to see if anything was wrong with it. This is on a Raspberry Pi and I have turned off ipv6 because that won't affect anything in my internal network. But I did that probably over a year ago and I've been using pianobar on it for a very long time without issue. No settings changed; this just started happening out of the blue. I was curious if others have been seeing it recently as well since my setup hasn't changed at all but pandora has been a bit weird for me lately.  

I'm also not getting the "trying next address" error. I'm getting the "connection timed out". My connection to everything else is fine so I'm not sure what's happening here. I was hoping others might be experiencing it.
